src/mmw/mmw/settings/development.py

```diff
@@ -71,3 +71,6 @@
 }
 
 # END TOOLBAR CONFIGURATION
+
+# API key for testing/development
+GOOGLE_MAPS_API_KEY = 'AIzaSyB0D5gjoIHpmy-xdP2cr_0I-E7K6s_L0k4'
```

Comment:
I did some googling around, and people go back and forth on whether or not this should be checked in. Since it's exposed on the client-side, it doesn't seem to be as big of a deal, but I'll defer to someone with more web security knowledge.